### PR TITLE
IRSA-1743_fix_msx

### PR DESCRIPTION
**Pull request by deannaji**
Kindly review:
This ticket is about changing MSX from using Oasis to Atlas api, using "irsa-image-master-table.csv" file.
 
Jira: https://jira.ipac.caltech.edu/browse/IRSA-1743
K8s: https://irsawebdev9.ipac.caltech.edu/irsa-1743/irsaviewer/

To test: Select an image position (for example m16), and check MSX.

### CODE REVIEW

File: src/firefly/java/edu/caltech/ipac/firefly/resources/irsa-image-master-table.csv [hunk 1]
```diff
@@ -26,6 +26,10 @@ AKARI,AKARI Far-Infrared All-sky Maps,,akari.akari_images,AKARI FIS N60,N60,,FIS
 AKARI,AKARI Far-Infrared All-sky Maps,,akari.akari_images,AKARI FIS N160,N160,,FIS N160 (160 microns),160,160 microns,far-IR,all-sky,all-sky,0.08,2,http://irsa.ipac.caltech.edu/data/AKARI/overview.html,Far-Infrared All-sky Maps N160 (160 microns),ATLAS,file_type='science',image
 AKARI,AKARI Far-Infrared All-sky Maps,,akari.akari_images,AKARI FIS WIDES,WideS,,FIS WideS (90 microns),90,90 microns,far-IR,all-sky,all-sky,0.08,2,http://irsa.ipac.caltech.edu/data/AKARI/overview.html,Far-Infrared All-sky Maps WIDE-S (90 microns),ATLAS,file_type='science',image
 AKARI,AKARI Far-Infrared All-sky Maps,,akari.akari_images,AKARI FIS WIDEL,WideL,,FIS WideL (140 microns),140,140 microns,far-IR,all-sky,all-sky,0.08,2,http://irsa.ipac.caltech.edu/data/AKARI/overview.html,Far-Infrared All-sky Maps WIDE-L (140 microns),ATLAS,file_type='science',image
+MSX,MSX,,msx.msx_images,MSX A (8.28 microns),A,,MSX A (8.28 microns),8.28,8.28 microns,far-IR,all-sky,all-sky,0.08,2,http://irsa.ipac.caltech.edu/data/MSX/overview.html,MSX A,ATLAS,file_type='science',image
```
Comment (ejoliet) [inline src/firefly/java/edu/caltech/ipac/firefly/resources/irsa-image-master-table.csv]: For the title, i'd change it to:  'MSX: Midcourse Space Experiment'

Comment (deannaji) [reply src/firefly/java/edu/caltech/ipac/firefly/resources/irsa-image-master-table.csv]: Sorry, I completely forgot about it :)

Comment (ejoliet) [reply src/firefly/java/edu/caltech/ipac/firefly/resources/irsa-image-master-table.csv]: Also you should ask Vandana about what is the waveband category (far-ir?) and project type (all-sky?)

Comment (deannaji) [reply src/firefly/java/edu/caltech/ipac/firefly/resources/irsa-image-master-table.csv]: Done, thanks!

**Pull request merged**